growth.md

```diff
@@ -0,0 +1,54 @@
+# Growth
+Growth is a multimodal, very brittle thing. Every time we start focusing on something, all the other channels suffer, but ultimately, making the company grow is one of the main target of the whole organization.
```

Comment:
Remove "very brittle things... channels suffer". Sounds pessimistic, nor it's true.

In fact, success in one channel helps push sucess in another one. Eg: A blog post (content marketing) brings both traffic from Reddit and HN (social media), plus organic search (SEO).

Reply:
Thanks!
Point being pushing people to be careful but I get the vibe.

Reply:
Ahhh gotcha